src/connectors/connector_cloudtasks.workflows.json

```diff
@@ -15,10 +15,10 @@
           "task_id": "example-task-id"
         },
         {
-          "schedule_time": "[fill this in]"
+          "schedule_time": "TIME_IN_ZULU_FORMAT"
```

Comment:
nit, is the user expected to fill something in here? similar to before, its not clear if this is a standard enum or a text field. If they are, we should add a comment w/ an example

Reply:
Ack. Unfortunately, we can't have comments in the JSON files.